backend/pick-git/src/main/java/com/woowacourse/pickgit/config/OAuthConfiguration.java

```diff
@@ -23,55 +33,59 @@ public OAuthConfiguration(OAuthService oAuthService) {
+        );
+    }
+
     @Override
     public void addArgumentResolvers(List<HandlerMethodArgumentResolver> resolvers) {
         resolvers.add(authenticationPrincipalArgumentResolver());
     }
 
     @Override
     public void addInterceptors(InterceptorRegistry registry) {
-        HandlerInterceptor authenticationInterceptor = new PathMatchInterceptor(
-            authenticationInterceptor())
-            .addPathPatterns("/api/posts/me", HttpMethod.GET)
-            .addPathPatterns("/api/github/repositories", HttpMethod.GET)
-            .addPathPatterns("/api/github/search/repositories", HttpMethod.GET)
-            .addPathPatterns("/api/github/repositories/*/tags/languages", HttpMethod.GET)
-            .addPathPatterns("/api/posts", HttpMethod.POST)
-            .addPathPatterns("/api/posts/*/likes", HttpMethod.PUT, HttpMethod.DELETE)
-            .addPathPatterns("/api/posts/*/comments", HttpMethod.POST)
-            .addPathPatterns("/api/profiles/me", HttpMethod.GET, HttpMethod.POST)
-            .addPathPatterns("/api/profiles/me/*", HttpMethod.PUT)
-            .addPathPatterns("/api/profiles/*/followings", HttpMethod.POST, HttpMethod.DELETE)
-            .addPathPatterns("/api/posts/*", HttpMethod.PUT, HttpMethod.DELETE)
-            .addPathPatterns("/api/profiles/*/contributions", HttpMethod.GET);
-
-        HandlerInterceptor ignoreAuthenticationInterceptor = new PathMatchInterceptor(
-            ignoreAuthenticationInterceptor())
-            .addPathPatterns("/api/profiles/*", HttpMethod.GET)
-            .addPathPatterns("/api/posts", HttpMethod.GET)
-            .addPathPatterns("/api/posts/*", HttpMethod.GET)
-            .addPathPatterns("/api/search/**", HttpMethod.GET)
-            .addPathPatterns("/api/profiles/*/followings", HttpMethod.GET)
-            .addPathPatterns("/api/profiles/*/followers", HttpMethod.GET)
-            .excludePatterns("/api/profiles/*/followings", HttpMethod.POST, HttpMethod.DELETE)
-            .excludePatterns("/api/profiles/me", HttpMethod.GET)
-            .excludePatterns("/api/posts/me", HttpMethod.GET)
-            .addPathPatterns("/api/posts/*/comments", HttpMethod.GET);
+        PathMatchInterceptor authenticationInterceptor =
+            new PathMatchInterceptor(authenticationInterceptor());
+        PathMatchInterceptor ignoreAuthenticationInterceptor =
+            new PathMatchInterceptor(ignoreAuthenticationInterceptor());
+
+        var autoAuthorizationInterceptorRegister = AutoAuthorizationInterceptorRegister.builder()
+            .storageForRegisterTypes(getStorageForRegisterTypes())
+            .authenticationInterceptor(authenticationInterceptor)
+            .ignoreAuthenticationInterceptor(ignoreAuthenticationInterceptor)
+            .uriParser(getUriParser())
+            .build();
+
+        autoAuthorizationInterceptorRegister.execute();
+        ignoreAuthenticationInterceptor.excludePatterns("/api/profiles/me", HttpMethod.GET);
```

Comment:
```suggestion
        ignoreAuthenticationInterceptor
            .excludePatterns("/api/profiles/me", HttpMethod.GET);
```

추후 가독성을 위해 개행을 한번 해주는 게 좋을 거 같아요!